bigquery/google/cloud/bigquery/job.py

```diff
@@ -2876,6 +2876,29 @@ def _format_for_exception(query, job_id):
 
         return template.format(job_id=job_id, header=header, ruler=ruler, body=body)
 
+    def _begin(self, client=None, retry=DEFAULT_RETRY):
+        """API call:  begin the job via a POST request
+
+        See
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/insert
+
+        :type client: :class:`~google.cloud.bigquery.client.Client` or
```

Comment:
Nit: We're using Google-style docstrings for new functions.

```suggestion
        Args:
            client (Optional[google.cloud.bigquery.client.Client]):
                The client to use. If not passed, falls back to the ``client``
                associated with the job object.
```

Style guide: http://google.github.io/styleguide/pyguide.html#383-functions-and-methods (We follow PEP-8 for everything except docstrings, though.)

See examples at https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google

Reply:
I also noticed that, but then realized the method was based on one of the existing methods, and that method also uses Sphinx-style docstrings.

If changing the docstring to Google style, I suggest doing the same in the original parent class's method as well.

Reply:
Ticket - https://github.com/googleapis/google-cloud-python/issues/9092